### DISCUSSION TITLE
Is there way to set animation speed factor?

### DISCUSSION THREAD

**Discussion by ichenhe** [coil-kt/coil#1313] [Q&A] [answered]
I am using coil with jetpack compose. It works great.

Now, I want to set a speed factor to animation (gif etc.) to speed up or slow down the animation.

Any suggestion? Thanks.

**Comment by Jawnnypoo** [comment, marked as answer]
Hey there, playback animation speed is not currently supported in GIFs, feel free to make a PR to add it!
